### PR TITLE
reduce locoglobals - _playerCompany

### PR DESCRIPTION
**Pull request by LeftofZen**

### CODE REVIEW

**Comment by AaronVanGeffen** [comment]
Maybe the access to all of these should pass through the CompanyManager instead of accessing the GameState directly.

File: src/OpenLoco/Windows/Construction/Common.cpp [hunk 1]
```diff
@@ -738,7 +739,7 @@ namespace OpenLoco::Ui::Windows::Construction
         // 0x0049EFEF
         static void drawRoadTabs(Window* self, Gfx::Context* context)
         {
-            auto company = CompanyManager::get(_playerCompany);
+            auto company = CompanyManager::get(getGameState().playerCompanies[0]);
```
Comment (duncanspumpkin) [inline src/OpenLoco/Windows/Construction/Common.cpp:742]: we already have a `getGameState().playerCompanies[0]` with `CompanyManger::getControllingId()` could also use `getPlayerCompany` which is a helper function for this.

**Review by AaronVanGeffen — APPROVED** [review]
Looks good to me :)

**Comment by LeftofZen** [comment]
> Looks good to me :)

thanks, more to come soon :)

**Pull request merged**